### PR TITLE
Change email template and date formats

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Add the name of the class, start time, and end times to the body of the reminder emails.
- Changed the date format on the clients modal for a more recognizable and easier to read format.

### Testing
How did you confirm your changes worked? 
- Sent emails and checked to see if the appropriate changes were reflected in the email body.
- Ran the application locally and saw that the date format on the client modal was changed.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Email before | Email after | 
| --- | --- |
| Screen Shot 2021-05-11 at 12 54 11 PM | Screen Shot 2021-05-11 at 12 50 54 PM | 

| Modal date before | Modal date after | 
| --- | --- |
| Screen Shot 2021-05-11 at 12 45 58 PM | Screen Shot 2021-05-11 at 12 44 13 PM |

### CODE REVIEW

**Review by eliasfang — APPROVED** [review]
This looks great! I noticed the date before and date after images are different dates/attendees but I'm assuming that's because they're different events.
I also got one of your test emails fo I know that works :)

File: app/views/emailTemplates/reminderEmail.ejs [hunk 1]
```diff
@@ -131,7 +131,7 @@
                     <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
                         <tr>
                             <td bgcolor="#ffffff" align="left" style="padding: 20px 30px 40px 30px; color: #666666; font-family: 'Lato', Helvetica, Arial, sans-serif; font-size: 18px; font-weight: 400; line-height: 25px;">
-                                <p style="margin: 0;">Get ready and get pumped for your upcoming class! Make sure to join on time and have everything you need to have a successful and worthwhile session. And don't forget to have fun! 😃</p>
+                                <p style="margin: 0;">Get ready and get pumped for your upcoming <%= classInfo.class_name %> class! It starts at <%= classInfo.scheduled_time %> and ends at <%= classInfo.class_end_time %>. Make sure to join on time and have everything you need to have a successful and worthwhile session. And don't forget to have fun! 😃</p>
```
Comment (mingkaili) [inline app/views/emailTemplates/reminderEmail.ejs]: Can you remove this emoji? I think google mail update its policy if there are emojis in email body, the email is highly likely to go to spam

**Review by mingkaili — CHANGES_REQUESTED** [review]

File: app/views/emailTemplates/reminderEmail.ejs (hunk 1 above)
Comment (shravan1k) [reply app/views/emailTemplates/reminderEmail.ejs]: removed

**Review by mingkaili — APPROVED** [review]
LTGM

**Pull request merged**